pyproject.toml

```
@@ -39,8 +39,8 @@ test-command = [
 environment.PIP_ONLY_BINARY = ":all:"
 
 [tool.cibuildwheel.linux]
-manylinux-x86_64-image = "manylinux1"
-manylinux-i686-image = "manylinux1"
+manylinux-x86_64-image = "manylinux2014"
+manylinux-i686-image = "manylinux2014"
```

```suggestion
manylinux-x86_64-image = "manylinux2010"
manylinux-i686-image = "manylinux2010"
```

I think for now let's bump the target up to manylinux2010, which will match what several other build related tools on PyPI are using (CMake + Ninja).